qa/integration/specs/monitoring_api_spec.rb

```diff
@@ -245,6 +245,33 @@
     logging_get_assert logstash_service, "INFO", "TRACE"
   end
 
+  it "should retrieve the pipeline flow statuses" do
+    logstash_service = @fixture.get_service("logstash")
+    logstash_service.start_with_stdin
+    logstash_service.wait_for_logstash
+    number_of_events.times { logstash_service.write_to_stdin("Testing flow metrics") }
```

Comment:
may be putting some sleep gives a window to flow metrics be captured?
```suggestion
    number_of_events.times {
      logstash_service.write_to_stdin("Testing flow metrics")
      sleep(1)
    }
```